Specification/4._CAN_bus_transport_layer.md

```diff
@@ -19,43 +19,39 @@ UAVCAN defines the following types of transfers:
 Both message and service transfers can be further distinguished between:
 
 * Single-frame transfer - a transfer that is entirely contained in a single CAN frame.
-* Multi-frame transfer - a transfer whose payload is distributed over multiple CAN frames.
+* Multi-frame transfer - a transfer that has its payload distributed over multiple CAN frames.
```

Comment:
"Who" is for people. That is generally for everything else, but can sometimes be used for people.